tools/buildmgr/cbuildgen/src/Console.cpp

```diff
@@ -155,6 +156,7 @@ int main(int argc, const char *argv[])
     else if  (arg.compare("--quiet")                 == 0)            quiet = true;
     else if  (arg.compare(0, 12, "--pack_root=")     == 0)            packRoot = arg.substr(12, arg.length());
     else if  (arg.compare(0, 16, "--compiler_root=") == 0)            compilerRoot = arg.substr(16, arg.length());
+    else if  (arg.compare("--no-update-rte")         == 0)            updateRteFiles = false;
```

Comment:
We agreed to have an inverted logic for `cbuildgen`, as commented [here](https://github.com/Open-CMSIS-Pack/devtools/pull/432#issuecomment-1273476662):
> in other words by default don't create/update RTE files but introduce an `--update-rte` flag

Please rework this behaviour, or if you prefer to provide it in a separated PR please remove the "--no-update-rte" option here.